fp_app/spec/factories/schedules.rb

```diff
@@ -0,0 +1,13 @@
+FactoryBot.define do
+  factory :schedule do
+    planner
+    started_at { Time.current+1.day }
+    is_available { false }
+  end
+
+  factory :schedule_with_appointments, parent: :schedule do
+    planner
+    started_at {Time.parse("2024-09-30 17:00:00")}
```

Comment:
この辺もrubocop動かせば自動で直してくれると思います
```suggestion
    started_at { Time.parse("2024-09-30 17:00:00") }
```